<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCEAP</class>
 <widget class="QMainWindow" name="CCEAP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CCEAP-Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="ip_lineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If runnung both client and server on the same computer, use&lt;/p&gt;&lt;p&gt;127.0.0.1 for servers ip&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>default is: 127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="port_lineEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>default port is 4444</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="startServerButton">
         <property name="text">
          <string>start server</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QListView" name="listView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_CCEAP"/>
    <addaction name="actionDeveloper"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_CCEAP">
   <property name="text">
    <string>about CCEAP</string>
   </property>
  </action>
  <action name="actionDeveloper">
   <property name="text">
    <string>developer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
